### PR TITLE
fixes-#6961-enforce java code styling

### LINKED ISSUE #6961: enforce java code styling (similar to prettier on frontend)
we could e.g. use 

https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html

There's also 

https://github.com/jhipster/prettier-java

The idea would be to run this as part of continuous integration. Ideally on CircleCI, see:

https://github.com/cBioPortal/cbioportal/blob/master/.circleci/config.yml

On the frontend we are running prettier already so we can try to do something similar (search for the prettier rule):

https://github.com/cBioPortal/cbioportal-frontend/blob/master/.circleci/config.yml

Over there we are running it only on the files changed in the PR, but if that's too complicated and running it on all the files is fast enough (e.g. a few minutes) we could also run it on all the files instead.

CC: @sheridancbio @n1zea144

### PR DESCRIPTION
**Pull request by rahul799**
Fix #6961 

Describe changes proposed in this pull request:
- Running prettier-Java to make a standard code format.

# Checks
- [ ] Runs on heroku
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

### CODE REVIEW

**Comment by rahul799** [comment]
@inodb any idea, why Travis is taking so long ?

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,14 @@ cache:
...
-  - MAVEN_VERSION=3.5.4 TEST=sanity-checks
-  - MAVEN_VERSION=3.5.4 TEST=end-to-end
+  - 
+  =3.5.4 TEST=core
+  - 
+  =3.5.4 TEST=python-validator
+  - 
+  =3.5.4 TEST=sanity-checks
+  - 
+  =3.5.4 TEST=end-to-end
```
Comment (inodb) [inline .travis.yml]: @rahul799 i think travis is not working because of these changes? If you revert them it might work

Comment (inodb) [reply .travis.yml]: or did u edit it after it stopped working?

Comment (rahul799) [reply .travis.yml]: Hope Travis won't fail this time. :crossed_fingers:

**Comment by inodb** [comment]
@rahul799 sorry about the travis failures. You can ignore those for now. Please go ahead and see if you can add a job to the .circleci/config.yml file that runs prettier. Thank you!

**Comment by rahul799** [comment]
@inodb ok.

**Comment by rahul799** [comment]
@inodb I am entirely new with the .yaml files & build tools, but I will fix the issues.

**Pull request closed without merging**